HW22/roles/hw221/tasks/adduserloop.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- name: add groups
+  become: true
+  group:
+    name: "{{item}}"
+    state: present
+  loop: "{{user_groups}}"
+
+- name: add users using loop
+  become: true
+  user:
+    name: "{{item}}"
+    home: "{{user_home}}/{{item}}"
+    groups: "{{user_groups}}"
+    shell: "{{user_shell}}"
+    state: present
+    password: "{{password | password_hash('sha512')}}"
+  loop:
```

Comment:
Each user should have it own parameters. Please use list of parameters to create user